azure-pipelines.yml

```diff
@@ -15,25 +6,31 @@ pr:
 - master
 
 # Three phases for each of the three OSes we want to run on
-phases:
-- template: /eng/common/templates/phases/base.yml
+jobs:
+- template: /eng/common/templates/job/job.yml
   parameters:
     name: Windows_NT
+    enableMicrobuild: true
+    enablePublishBuildArtifacts: true
+    enablePublishTestResults: true
+    enablePublishBuildAssets: true
     enableTelemetry: true
-    queue:
+    helixRepo: dotnet/arcade-minimalci-sample
+    pool:
       ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
         name: dotnet-external-temp
       ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
         name: dotnet-internal-temp
-      parallel: 99
+    strategy:
       matrix:
         debug_configuration:
           _BuildConfig: Debug
         release_configuration:
           _BuildConfig: Release
-    variables:
-      _HelixBuildConfig: $(_BuildConfig)
     steps:
+    - checkout: self
+      clean: true
+
```

Comment:
why do you need the nuget commands below rather than just using the cli restored by arcade to do the package restore?

Reply:
The "dotnet restore" actually crashes if you try to restore it, because it weirdly requires the package to already be restored. The "Service Fabric" projects that VS creates are... very strange.

Reply:
I think it's the fact that service fabric projects _MUST_ use the nuget.config method of package specifications (the SDK will force that if you try to avoid it), and dotnet doesn't understand that?